aws-provider/postgresdb/README.md

```diff
@@ -0,0 +1,346 @@
+   this policy to it. The Open ID Connect Identity Provider should be set to the
+   one for your cluster.
+
+   - substitute the variables `AWS_ACCOUNT_ID` and `OICD_PROVIDER_URL` in the
+     trust policy:
+
+     ```bash
+      export OIDC_PROVIDER_URL=$(aws iam list-open-id-connect-providers \
+          | jq '.OpenIDConnectProviderList[] | .Arn | split("/")[1] | select(. | test("^\\w+.mycluster.(?!mycluster)"))')
+      export AWS_ACCOUNT_ID=$(aws sts get-caller-identity | jq -rn .Account)
+      cat policies/crossplane-assume-role-trust-policy.json | envsubst \
+          | sponge policies/crossplane-assume-role-trust-policy.json
+     ```
+
+   - Check that the role does not already exist
+
+     ```bash
+     aws iam list-roles | jq '.Roles[] | select( .RoleName == "crossplane-assume-role")'
+     ```
+
+     If the role does not exist, we need to create it and attach the policy
+
+     ```bash
+       aws iam create-role --role-name crossplane-assume-role --policy-document file://policies/crossplane-assume-role-trust-policy.json
+       aws iam attach-role-policy --policy-arn \
+           "arn:aws:iam::aws:policy/crossplane-assume-role-policy" \
+           --role-name crossplane-assume-role
+     ```
+
+     If the role already exists, update its trust policy with
+
+     ```bash
+       aws iam update-assume-role-policy --role-name crossplane-assume-role \
+           --policy-document file://policies/crossplane-assume-role-trust-policy.json
+     ```
+
+## Create the secondary role
+
+Use the steps defined in [creating the primary role](#create-the-primary-role) to
+create a new role `rds-crossplane-role`
+
+This second role will use the attach the trust policy and IAM permissions policy
+
+- [rds-crossplane-policy](./policies/rds-crossplane-policy.json).
+- [rds-crossplane-role-trust-policy](./policies/rds-crossplane-role-trust-policy.json)
+
+> **Note** Creating IAM policies to be assumed by other roles
+>
+> When creating roles to be assumed by the service account identity role
+> `crossplane-assume-role`, at the very least, the policy must include the
+> following statement for other roles to be allowed to assume it.
+>
+> ```json
+> {
+>     "Sid": "VisualEditor2",
+>     "Effect": "Allow",
+>     "Action": "sts:AssumeRole",
+>     "Resource": "*",
+>     "Condition": {
+>         "ForAnyValue:StringLike": {
+>             "aws:PrincipalArn": [
+>                 "arn:aws:sts::${AWS_ACCOUNT_ID}:role/crossplane-assume-role",
+>                 "arn:aws:sts::${AWS_ACCOUNT_ID}:assumed-role/rds-crossplane-role*"
+>             ]
+>         }
+>     }
+> }
+> ```
+>
+> Without this, crossplane may not be able to assume the role and you will
+> experience permissions failures when building resources.
+
+The policy file [rds-crossplane-policy](./policies/rds-crossplane-policy.json)
+is a restrictive policy that allows only the specific actions required to build
+this example.
+
+## Enabling providers
+
```

Comment:
Maybe a short intro, "Since new crossplane versions come without any API enabled, we need to add every AWS family/service to Crossplane in order to...."

Reply:
I would disagree with this wording. New crossplane versions do not have a concept of an API and don't remove providers; similarly, the wording "We need to add every AWS family/service to Crossplane..." would imply that we're going to install every family member, all 159 of them.

This would offer even worse performance to the cluster than just installing `provider-aws` as this would pull in 159 additional deployments, each scalable to `n` replicas, this would be on top of the 903 CRDs it would also be installing.

Maybe something closer to this?

> By default, crossplane comes without any providers enabled and as such, we need to install the providers we're going to use for building and managing our infrastructure.
> 
> In this exampled we require the following providers to be enabled.
> 
> - EC2
> - RDS
> - KMS
> 
> The name of each provider may be discovered from its API group prefixed with `xpkg.upbound.io/provider-aws-`. 
> 
> For example, for `ec2.aws.upbound.io` becomes `xpkg.upbound.io/provider-aws-ec2@VERSION` where `VERSION` is the exact provider version you wish to install, discoverable from [doc.crds.dev/github.com/upbound/provider-aws](https://doc.crds.dev/github.com/upbound/provider-aws)

Reply:
that sounds good to me ;)